vue-app/src/components/IpfsCopyWidget.vue

```diff
@@ -111,20 +109,6 @@ export default class IpfsCopyWidget extends Vue {
 }
 
 .icon {
-  width: 1rem;
-  height: 1rem;
-  padding: 0.25rem;
-  cursor: pointer;
-  &:hover {
-    background: $bg-light-color;
-    border-radius: 16px;
-  }
-}
-
-.hash-text {
```

Comment:
I noticed through testing we lost this styling functionality where the ellipsis show: 
<img width="271" alt="Screen Shot 2021-07-09 at 8 25 22 PM" src="https://user-images.githubusercontent.com/15589226/125149083-04acaf00-e0f4-11eb-9974-7b41f38a2cda.png">

I think we should try and get this back in with this PR